src/geff/metadata_schema.py

```diff
@@ -235,7 +242,7 @@ class GeffMetadata(BaseModel):
     )
 
     geff_version: str = Field(
-        ...,
+        default=GEFF_VERSION,
```

Comment:
see `formatted_schema_json` below as a new approach to declare in the schema that geff_version is required.
This does otherwise pose typing problems for anyone trying to create a GeffMetadata without passing version.  (we were lying about the default in order to satisfy the schema)